ChessCloudCode/Properties/PublishProfiles/FolderProfile.pubxml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+https://go.microsoft.com/fwlink/?LinkID=208121. 
+-->
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <Configuration>Release</Configuration>
+    <Platform>Any CPU</Platform>
+    <PublishDir>bin\Release\net5.0\publish\</PublishDir>
+    <PublishProtocol>FileSystem</PublishProtocol>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
`net5.0`? Should that not be net7.0? https://github.com/Unity-Technologies/com.unity.services.samples.multiplayer-chess-cloud-code/blob/main/ChessCloudCode/ChessCloudCode.csproj#L4 (does it matter?)

Reply:
Mistake. That's unfortunately the default that the CLI generates atm with new-file (they're aware and changing that), though I was sure I fixed it.